developer-docs-site/docs/tutorials/full-node/network-identity-fullnode.md

```diff
@@ -6,10 +6,20 @@ sidebar_position: 12
 
 # Network Identity For FullNode
 
+FullNodes will automatically start up with a randomly generated network identity. This works well for regular FullNodes. However:
+
+- You may want your FullNode to be added to a specific upstream FullNode's allowlist (i.e., other FullNode participant in the Aptos network), because:
+
+  - You might require specific permissions for your FullNode on this specific upstream FullNode, or
+  - This upstream FullNode only allows known identities to connect to it.
+
+In such cases, it helps if you run your FullNode with a static network identity, instead of a randomly generated network identity that keeps changing every time you start up your FullNode.
+
 This guide will show you how to:
+
 - Create a static network identity for your FullNode.
-- Retrieve the public network identity.
-- Start a node with (or without) a static network identity.
+- Start a node with a static network identity.
+- Allow other FullNodes to connect to your Fullnode.
```

Comment:
nit: "Fullnode" -> "FullNode".

I really wish we hadn't normalized "FullNode" 😵‍💫